kong/runloop/plugins_iterator.lua

```diff
@@ -63,6 +64,12 @@ local function load_plugin_from_db(key)
     return nil, tostring(err)
   end
 
+  if type(row) == 'table' then
+    row.__key__ = key
+    row.__seq__ = next_seq
```

Comment:
The fact that this metadata is stored in-band with the plugin object table irks me a bit, but its use up there is pretty logical and this approach is generally more trouble-free than alternatives (weak tables, etc.). Given that only the `config` subtable is exposed to plugins, this means that these extra fields remain confined in the core, so that should be ok.